Library/Formula/cycript.rb

```diff
@@ -0,0 +1,91 @@
+class Cycript < Formula
+  desc "Modify apps using a hybrid of Objective-C++ and JavaScript syntax."
+  homepage "http://www.cycript.org"
+  url "git://git.saurik.com/cycript.git",
+    :tag => "v0.9.592",
+    :revision => "97bec96b43b66ab78af95d2b6c6c24f0d0a8006a"
+
+  head "git://git.saurik.com/cycript.git"
+
+  depends_on "bison" => :build
+  depends_on "android-sdk" => :build
+  depends_on "readline"
+
+  patch :DATA
+
+  def install
+    ENV.append "CPPFLAGS", "-Wno-inconsistent-missing-override"
+
+    system "./configure",
+           "--prefix=#{prefix}",
+           "CPPFLAGS=#{ENV.cppflags}"
+
+    system "make", "install"
+  end
+
+  test do
+    assert_equal "7", pipe_output("#{bin}/cycript -", "print([@[@2,@3].map(a => a + 1) valueForKeyPath:@\"@sum.self\"])").strip
+  end
+end
+__END__
+diff --git a/Execute.cpp b/Execute.cpp
+index 18bdce6..cb850f1 100644
+--- a/Execute.cpp
++++ b/Execute.cpp
+@@ -2153,7 +2153,7 @@ void CYInitializeDynamic() {
+     else return;
+ 
+     CYPool pool;
+-    const char *db(pool.strcat(CYPoolLibraryPath(pool), "/libcycript.db", NULL));
++    const char *db(pool.strcat(CYPoolLibraryPath(pool), "/../share/libcycript.db", NULL));
```

Comment:
Is this something that’ll change on version bumps? I mean if you update `cycript`, do you expect this file to be overrided by the new version or should that be something we store outside of the prefix?